ViewDeckExample.xcodeproj/project.pbxproj

```diff
@@ -343,7 +376,7 @@
 				GCC_PRECOMPILE_PREFIX_HEADER = YES;
 				GCC_PREFIX_HEADER = "ViewDeckExample/ViewDeckExample-Prefix.pch";
 				INFOPLIST_FILE = "ViewDeckExample/ViewDeckExample-Info.plist";
-				IPHONEOS_DEPLOYMENT_TARGET = 4.3;
+				IPHONEOS_DEPLOYMENT_TARGET = 7.0;
```

Comment:
This is a breaking change. This can not happen without a new major release. Is there a way for you to fix the issues without requiring iOS 7 API? Otherwise I would keep this PR on hold for the next major release.